spoutbreeze-commons/src/main/java/org/spoutbreeze/commons/enums/AgentStatus.java

```diff
@@ -19,5 +19,5 @@
 package org.spoutbreeze.commons.enums;
 
 public enum AgentStatus {
-    ENABLED, DISABLED
+    ENABLED, DISABLED, BROADCASTING

```

Comment:
We don't need the `BROADCASTING` status for the agent, the reason is that we can find it from `broadcasts` table and we will need more config later for selenoid, an agent can example have 5 broadcasts.

Reply:
Sure, makes sense.